adbms/src/main/scala/de/up/hpi/informationsystems/adbms/definition/Record.scala

```diff
@@ -37,20 +37,11 @@ class Record private (cells: Map[UntypedColumnDef, Any])
     * @param columnDefs columns to project to
     * @return A new record containing only the specified columns
     */
-  def project(columnDefs: Seq[UntypedColumnDef]): Try[Record] = Try(internal_project(columnDefs))
-
-  /**
-    * Iff all columns of the relation are a subset of this record,
-    * returns a new record with only the columns of the relation,
-    * otherwise returns an error message.
-    * @param r Relation to project this Record to
-    * @return A new record containing only the specified columns
-    */
-  def project(r: Relation): Try[Record] = Try(internal_project(r.columns))
```

Comment:
big +1 on removing this after reading you s/o-link!

Reply:
pls also remove `Relation#project(r: Relation): Seq[Record]` in Relation.scala and all implementations

Reply:
I'm doing this on this branch now. This way I can write the TestApplication correctly once here as well and we don't have to merge PRs in a specific order. Hope you're alright with this.